src/electron/views/device-connect-view/components/android-setup/android-setup-spinner-layout.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import { AndroidSetupActionCreator } from 'electron/flux/action-creator/android-setup-action-creator';
+import {
+    DeviceConnectFooter,
+    DeviceConnectFooterDeps,
+} from 'electron/views/device-connect-view/components/device-connect-footer';
+import { Spinner, SpinnerSize } from 'office-ui-fabric-react';
+import * as React from 'react';
+
+import { androidSetupSpinner } from './android-setup.scss';
+
+export type AndroidSetupSpinnerDeps = {
+    androidSetupActionCreator: AndroidSetupActionCreator;
+} & DeviceConnectFooterDeps;
+export type AndroidSetupSpinnerProps = {
+    deps: AndroidSetupSpinnerDeps;
+    spinnerLabel: string;
+};
+
+export const AndroidSetupSpinnerLayout = NamedFC<AndroidSetupSpinnerProps>(
+    'AndroidSetupSpinnerLayout',
+    (props: AndroidSetupSpinnerProps) => {
+        return (
+            <>
+                <Spinner
+                    className={androidSetupSpinner}
+                    size={SpinnerSize.large}
+                    label={props.spinnerLabel}
+                    role="alert"
+                    aria-live={props['aria-live']}
+                />
+                <DeviceConnectFooter
```

Comment:
I think we decided in the feature chat that we should use "Next" rather than the `DeviceConnectFooter`'s "Start testing" for the disabled bottom-right button. You could achieve this by updating the `DeviceConnectFooter` to take a a label param; however, I chose not to do that for the `AndroidSetupPromptLayout`, since I thought repeating a bit of the impl now would make it easier to remove the obsolete components along with the feature flag at the end of the feature.

That said, it does seem like it would be a good idea to share the wrapping `<main>` element and the footer button layout between the spinner and prompt layouts; maybe sharing a "base layout" between this class and the prompt layout would be the way to go? I'm imagining splitting out an `AndroidSetupBaseLayout` from part of the current `AndroidSetupPromptLayout` such that this `AndroidSetupSpinnerLayout` might look something like:

```ts
return (<AndroidSetupBaseLayout {...props}>
    <Spinner
        className={androidSetupSpinner}
        size={SpinnerSize.large}
        label={props.spinnerLabel}
        role="alert"
        aria-live="polite" />
</AndroidSetupBaseLayout>);
```